src/main/java/net/minecraftforge/client/event/GuiScreenEvent.java

```diff
@@ -159,6 +159,20 @@ public int getMouseY()
             return mouseY;
         }
     }
+
+    /**
+     * This event fires in {@code InventoryEffectRenderer.updateActivePotionEffects()}
```

Comment:
Fixing this documentation issue here: https://github.com/MinecraftForge/MinecraftForge/pull/2833